WordPress/Tests/Theme/NoCDNUnitTest.inc

```diff
@@ -0,0 +1,30 @@
+<?php
+// Now lets check inside php.
```

Comment:
Good call on having some examples with both `http` as well as `https`. What about other protocols ? Is `ftp` ever used ? Any other protocols you may have encountered ?

I realize that for the sniff it won't make any difference, but might be good to show that those cases (if they exist) are covered.